spec/services/dashboard_event_handler_spec.rb

```diff
@@ -113,4 +113,18 @@
       merits_assessment.submit!
     end
   end
+
+  context 'applicant_emailed' do
+    let(:legal_aid_application) { create :legal_aid_application, :with_applicant }
+    before do
+      # allow_any_instance_of(DashboardEventHandler).to receive(:firm_created)
+      # allow_any_instance_of(DashboardEventHandler).to receive(:provider_created)
+      # allow_any_instance_of(DashboardEventHandler).to receive(:application_created)
```

Comment:
Can these be removed?

Reply:
Good catch... they seemed pointless and the passing tests confirmed it! 😄